### PR TITLE
Improve grammar and spelling in Contributing Guide

### PR DESCRIPTION
**Pull request by adamchainz**
# Pull Request Check List

This is just a reminder about the most common mistakes. Please make sure that you tick all *appropriate* boxes.  But please read our [contribution guide](https://poetry.eustace.io/docs/contributing/) at least once, it will save you unnecessary review cycles!

- [x] Added **tests** for changed code.
- [x] Updated **documentation** for changed code.

**Note**: If your Pull Request introduces a new feature or changes the current behavior, it should be based
on the `develop` branch. If it's a bug fix or only a documentation update, it should be based on the `master` branch.

If you have *any* questions to *any* of the points above, just **submit and ask**!  This checklist is here to *help* you, not to deter you from contributing!

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # Contributing to Poetry
 
-First off, thank for taking the time to contribute!
+First off, thanks for taking the time to contribute!
 
-The following is a set of guidelines for contributing to Poetry on GitHub. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+The following is a set of guidelines for contributing to Poetry on GitHub. These are mostly guidelines, not rules. Use your best judgement, and feel free to propose changes to this document in a pull request.
 
-#### Table of Contents
+#### Table of contents
```
Comment (drunkwcodes) [inline CONTRIBUTING.md]: Remain [title case](https://titlecase.com)?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ Before creating bug reports, please check [this list](#before-submitting-a-bug-r
 * **Check the [FAQs on the official website](https://poetry.eustace.io)** for a list of common questions and problems.
 * **Check that your issue does not already exist in the [issue tracker](https://github.com/sdispater/poetry/issues)**.
 
-#### How do I submit a bug report
+#### How do I submit a bug report?
```
Comment (drunkwcodes) [inline CONTRIBUTING.md]: Remain title case here too?

File: CONTRIBUTING.md (hunk 1 above)
Comment (adamchainz) [reply CONTRIBUTING.md]: I removed title case to be consistent with all the other headings which are not

File: CONTRIBUTING.md (hunk 2 above)
Comment (adamchainz) [reply CONTRIBUTING.md]: I didn't change this, just added the missing question mark

**Comment by adamchainz** [comment]
Happy to convert all the headings to title case if that's the desired style

File: CONTRIBUTING.md (hunk 1 above)
Comment (drunkwcodes) [reply CONTRIBUTING.md]: Pardon me that I ignored it's English here.

**Comment by sdispater** [comment]
Thanks!

**Pull request merged**